src/LocationSelector/index.js

```diff
@@ -1,39 +1,43 @@
 import React from "react";
 import { geocodeByLatlng, getAddressComponent, isFunction } from "../utils";
-import MapWithSearch from "../MapWithSearch";
+import LocationSelectorMapWithSearch from "../MapsWithSearch/LocationSelectorMap";
 
 class LocationSelector extends React.Component {
+  geocodeAndInformParent = async latLng => {
+    const results = await geocodeByLatlng(latLng);
```

Comment:
What happens if this api call fails ? Will it break the page ?

Reply:
even if it doesn't i would suggest to add a try catch around it, and console log the error

Reply:
done